src/NHibernate.Test/Async/Legacy/FooBarTest.cs

```diff
@@ -5162,6 +5162,7 @@ public async Task EmbeddedCompositeIDAsync()
 			l.CountryCode = "AU";
 			l.Description = "foo bar";
 			l.Locale = CultureInfo.CreateSpecificCulture("en-AU");
+			l.ExtendedLocale = CultureInfo.CreateSpecificCulture("en-US-POSIX");
```

Comment:
There are no dedicated test for CultureInfoType. This one was testing it, along other things. So, I have added the test for the variable length case here. Otherwise a dedicated test in TypesTest could be added.

Reply:
I would probably prefer a dedicated test for the type.

Reply:
Done.